spec/services/edge_cache/bust_podcast_episode_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+RSpec.describe EdgeCache::BustPodcastEpisode, type: :service do
+  let(:podcast) { create(:podcast) }
+  let(:podcast_episode) { create(:podcast_episode, podcast_id: podcast.id) }
+  let(:podcast_path) { "/cfp" }
+  let(:podcast_slug) { "-007" }
+
+  let(:paths) do
+    [
+      podcast_path,
+      "/#{podcast_slug}",
+      "/pod",
+    ]
+  end
+
+  before do
+    paths.each do |path|
+      allow(described_class).to receive(:bust).with(path).once
+    end
+
+    allow(podcast_episode).to receive(:purge)
+    allow(podcast_episode).to receive(:purge_all)
+  end
+
+  it "busts the cache" do
+    described_class.call(podcast_episode, podcast_path, podcast_slug)
+
+    paths.each do |path|
+      expect(described_class).to have_received(:bust).with(path).once
+    end
+
+    expect(podcast_episode).to have_received(:purge)
+    expect(podcast_episode).to have_received(:purge_all)
+  end
+
+  it "logs an error" do
+    allow(Rails.logger).to receive(:warn)
+    allow(described_class).to receive(:bust).and_raise(StandardError)
+    described_class.call(podcast_episode, 12, podcast_slug)
```

Comment:
Same question about the `12` here :) The function definition makes me think we should be passing `podcast_path` here instead of `12`...but if not, I am curious why not

Reply:
The same thing here - I have no idea, I copied it over from the existing spec

https://github.com/forem/forem/blob/c2982ef0fc9dfc75a12ebdbfb8f78fc16110a8ac/spec/labor/cache_buster_spec.rb#L92

My guess is `12` is an "invalid argument".